include/coap2/coap_dtls.h

```diff
@@ -30,6 +30,16 @@ struct coap_dtls_pki_t;
 #define COAP_DTLS_HINT_LENGTH 128
 #endif
 
+/* https://tools.ietf.org/html/rfc4347#section-4.2.4.1 */
```

Comment:
The proper reference for DTLS1.2 is RFC 6347. (The referenced text would be the same, though.)

Reply:
Code updated and pushed